utils/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@microsoft/vscode-azext-utils",
     "author": "Microsoft Corporation",
-    "version": "0.2.5",
+    "version": "0.2.6",
```

Comment:
Technically a semver violation, but otherwise we have to update all the upstream packages to avoid version conflicts, and I don't really feel like doing that right now...

Reply:
Thanks to CG alerts we no longer have a choice.